data/human/news.txt

```diff
@@ -1616,6 +1753,27 @@ news "lawyer"
 			`"`
 
 
+news "convicted criminal"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	name
+		word
+			"Convict"
+			"Convicted Criminal"
+			"Transported Convict"
+	message
+		word
+			`"`
+		word
+			"I fought the law, and the law won."
+			"I'm innocent, I tell you! They convicted me for five murders, yet I only killed two people!"
+			"Two years of food and housing for free, and all the time in the world to read. I'm so looking forward to staying in prison."
+			"I should have gone to one of those pirate worlds. They would never have caught me."
+			"They said I took a bribe to lose the game, but that is not true! I would also have gotten the credits if we had won!"
+			"The rich just buy themselves out of legal troubles! The Mayor was in on the plan, but he got away with a fine of 100 credits and I'm here serving five years!"
```

Comment:
100 credits seems too low, given that a passenger transport is like 5k. Maybe 10000 credits?

Reply:
I will change it to 1000 credits, the Mayor still got away really easily. :)